CHANGELOG.unreleased.md

```diff
@@ -44,6 +44,16 @@ const link = <a href="example.com">http://example.com</a>;
 
 -->
 
+#### TypeScript: Support TypeScript 3.7 syntax features ([#6657] by [@Cryrivers])
+
+Add following syntax support for TypeScript 3.7:
+
+- Optional Chaining
+- Null Coalescing
+- `asserts` keyword
+
+NOTE: `declare` keyword on class members has not been supported yet, which will be fixed in future versions.
+
```

Comment:
Let's add examples for each feature as we do in CHANGELOG for other issues - Core/`Output (Prettier stable)`/`Output (Prettier master)`

Reply:
> Let's add examples for each feature as we do in CHANGELOG for other issues - Core/`Output (Prettier stable)`/`Output (Prettier master)`

Done :)